package.json

```
@@ -72,6 +72,8 @@
     "@react-navigation/stack": "6.3.17",
     "@sentry/react-native": "5.7.1",
     "@tradle/react-native-http": "2.0.1",
+    "@uniswap/sdk": "^3.0.3",
```

Please fix the version number
```suggestion
    "@uniswap/sdk": "3.0.3",
```